extensions/stac-eo-spec.md

```diff
@@ -82,6 +82,64 @@ numbers of several popular instruments.
 | lwir11      | 10.5 - 11.5     |           |           | 10        |            | 31    |
 | lwir12      | 11.5 - 12.5     |           |           | 11        |            | 32    |
 
+### Processing level fields
+
+We define **direct conversion** as a process that converts the input data to a physical measurement
+using only information available in the input data and its metadata. The process may be
+an identity function when data is already provided as the physical measurement.
+
+An Item is said to provide
+a processing level feature if there exists a direct conversion from data as presented to the
+physical measurement associated with the processing level.
+
+#### Radiometric processing features
+
+| Field Name                | Type                     | Description                                                                                                                                                                                                                                                |
+| ------------------------- | ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| eo:digital_number         | boolean                  | Data is a digital number that can not be converted to a physical quantity through a direct conversion                                                                                                                                                    |
+| eo:at_sensor_radiance     | boolean                  | At sensor radiance may be obtained by a direct conversion from original data.                                                                                                                                                     |
+| eo:at_sensor_reflectance  | boolean                  | At sensor reflectance may be obtained by a direct conversion from original data.                                                                                                                                                 |
+| eo:ground_reflectance     | boolean                  | Ground reflectance may be obtained by a direct conversion from original data.                                                                                                                                                     |
+
```

Comment:
There are other typical radiometric features such as relative calibration between detectors and interpolation of defective detectors. I did not include them because my feeling is that all datasets we've been considering for STAC always include this processing.